bedrock/externalpages/templates/externalpages/pocket/tos.html

```diff
@@ -0,0 +1,213 @@
+      <p>If you choose to create a user account, you are responsible for maintaining the confidentiality of your account and password and for restricting access to your account. You may not share your password or other account access information with any other party, temporarily or permanently, and you agree to accept responsibility for all activities that occur under your account or password, whether or not authorized by you. You agree to immediately notify us of any unauthorized use of your account, Registration Data or passwords, as the case may be. We reserve the right to refuse service and/or terminate user accounts in our sole discretion.</p>
+      <p>If you choose to send us any message, feedback or data, including, but not limited to, any ideas, comments, suggestions or questions regarding any product or service, such information shall be deemed to be non-confidential. We shall have no obligation of any kind with respect to such information and shall be free to reproduce, use, disclose and distribute the information to others without limitation. Further, we shall be free to use any ideas, concepts, know-how or techniques contained in such information for any purpose whatsoever, including but not limited to developing, manufacturing and marketing products and services incorporating such ideas, concepts, know-how or techniques. The Pocket Technologies enable users to share, save or post information or materials, or to communicate with or otherwise share information with other users or persons. You agree not to share, save, submit, publish, display, disseminate, or otherwise communicate any defamatory, inaccurate, abusive, threatening, offensive, indecent, illegal or otherwise objectionable material, or any material that violates any federal, state, or local law in the United States or anywhere else in the world. You are solely responsible for all information which you share, save, submit, publish, display, disseminate or otherwise communicate through the Pocket Technologies, even if a claim should arise after termination of service. You agree that all such content, messages and other communications shall not be deemed to be private or secure, and that any such content, messages and communications can be read by the operators and/or other agents of Mozilla, whether or not they are the intended recipients. If we or any other person asks you to cease and desist from sharing any videos, articles or other content with any third party, or to cease communicating with any third party via the Pocket Technologies, you agree to comply with such request immediately.</p>
+    </section>
+
+    <section id="copyright">
+      <h2>Copyright infringement</h2>
+      <p>We will keep your information only for as long as we need it to provide you services, manage our business or as required by law or contract.</p>
```

Comment:
This seems to be the wrong text seen here. The first paragraph is missing in this section, and I don't think that first text in the `p` tag is available on the original page?